perses/annihilation/relative.py

```diff
@@ -66,6 +66,8 @@ def __init__(self,
                  soften_only_new=False,
                  neglected_new_angle_terms = [],
                  neglected_old_angle_terms = [],
+                 special_new_terms = None,
```

Comment:
Should we use a term other than `special`? Do these represent biasing terms that are applied only during proposal, but are corrected for during the RJMC acceptance step?

Reply:
special terms is a dict of omitted bonds, angles, and torsions (which are annealed in through AIS), as well as extra angles and torsions that bias  RJ proposals (which are annealed off during the AIS protocol);